cmd/dep/status.go

```diff
@@ -49,6 +55,7 @@ func (cmd *statusCommand) Register(fs *flag.FlagSet) {
 	fs.BoolVar(&cmd.detailed, "detailed", false, "report more detailed status")
 	fs.BoolVar(&cmd.json, "json", false, "output in JSON format")
 	fs.StringVar(&cmd.template, "f", "", "output in text/template format")
+	fs.StringVar(&cmd.output, "o", "output.svg", "output file")
```

Comment:
`-dot` flag was in the code before: https://github.com/golang/dep/blob/master/cmd/dep/status.go#L52

This is a newly created flag for saving generated `dot` string into the file which is not necessary if we're stdout `string` now.